overlay/ports/ffmpeg/portfile.cmake

```diff
@@ -17,8 +17,74 @@ vcpkg_from_github(
         0040-ffmpeg-add-av_stream_get_first_dts-for-chromium.patch # Do not remove this patch. It is required by chromium
         0041-add-const-for-opengl-definition.patch
         0043-fix-miss-head.patch
+        # FIXME should be only applied if feature "fedora-ffmpeg-free-safe" is used
+        0099-ffmpeg-allow-fdk-aac-free.patch
```

Comment:
You can use a variable that is either empty or has the patch. 
The alternative using a dedicated vcpkg_apply_patches like here: 
https://github.com/microsoft/vcpkg/blob/11be7f536538ace43f867c8efb3d624c43b6af54/ports/ableton-link/portfile.cmake#L73
seems to be deprecated: 
https://github.com/microsoft/vcpkg/blob/11be7f536538ace43f867c8efb3d624c43b6af54/scripts/cmake/vcpkg_apply_patches.cmake#L2